docs/developer-guide.md

```diff
@@ -0,0 +1,41 @@
+# Developer Guide
+
+This is a very basic guide on how to set up your local environment, make the desired changes and see the result with a fresh Grafana Installation.
+
+## Getting Started
+Clone this repository into your local environment. The code lives in the `src` folder, alongside the [plugin.json file](https://grafana.com/docs/grafana/latest/developers/plugins/metadata/). See [this grafana tutorial](https://grafana.com/docs/grafana/latest/developers/plugins/) to understand better how a plugin is structured and installed.
+
+## Requirements
+
+For this standard execution, you will need the following tools:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [Yarn](https://classic.yarnpkg.com/en/docs/install)
+
+## Running the development version
+
+After you made the desired changes, you can build and test the new version of the plugin using `yarn`:
+
+```sh
+$ yarn test # run all test cases
+$ yarn dev # builds and puts the output at ./dist
+```
+
+Now that you have a `./dist` folder, you are ready to run a fresh Grafana instance and put the new version of the datasource into [Grafana plugin folder](https://grafana.com/docs/grafana/latest/plugins/installation/).
+
+We provide a [Docker Compose file](/docker-compose.yml) to help you to get started. When you call up `docker-compose up` inside the project folder, it will:
```

Comment:
```suggestion
### Docker Compose
We provide a [Docker Compose file](/docker-compose.yml) to help you to get started. When you call up `docker-compose up` inside the project folder, it will:
```